setup.py

```diff
@@ -155,6 +156,17 @@ def run(self):
                     run('ninja\\ninja -C out.gn/x64.release d8')
                 else:
                     gypflags = '-Dv8_use_external_startup_data=0 -Dv8_enable_i18n_support=0 -Dv8_enable_inspector=1 -Dwerror=\'\' '
+                    # for some reason, gtest will sometimes fail to build because the weird cmake-gyp bridge generates
+                    # clang calls that exclude the standard header path. we don't need it, skip building it
+                    run('sed -i "s|\'../samples/samples.gyp:\\*\',||g" gypfiles/all.gyp')
+                    run('sed -i "s|\'../test/cctest/cctest.gyp:\\*\',||g" gypfiles/all.gyp')
+                    run('sed -i "s|\'../test/fuzzer/fuzzer.gyp:\\*\',||g" gypfiles/all.gyp')
+                    run('sed -i "s|\'../test/unittests/unittests.gyp:\\*\',||g" gypfiles/all.gyp')
+                    run('sed -i "s|testing/gmock.gyp ||g" Makefile')
+                    run('sed -i "s|testing/gtest.gyp ||g" Makefile')
+                    run('sed -i "s|test/unittests/unittests.gyp ||g" Makefile')
+                    run('sed -i "s|test/cctest/cctest.gyp ||g" Makefile')
+                    run('sed -i "s|test/fuzzer/fuzzer.gyp ||g" Makefile')
```

Comment:
This seems incredibly brittle. That said, we're pinned to a really old V8 version (5.9, from Chrome 59, from 2017), so I don't expect it to break anytime soon. This project is definitely overdue for a V8 upgrade though.

Reply:
This was a workaround for an extremely strange bug when building inside the manylinux Docker image, which we no longer do. I've removed it, and I'm pretty sure it's not needed in Travis native since it was previously building fine.

Reply:
Cool!